libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meta, Story } from '@storybook/react';
+import { AnnouncementBanner } from '.';
+
+export default {
+  component: AnnouncementBanner,
+  title: 'Components/AnnouncementBanner',
+} as Meta;
+
+const Template: Story = (args) => {
+  return (
+    <AnnouncementBanner {...args}>
+      <div className="text-center text-xs font-semibold md:text-sm">
```

Comment:
This needs a `dark:text-black` class. Maybe the styles should be in the AnnouncementBanner?

Reply:
Also a good catch!